jetty-core/jetty-client/src/main/java/org/eclipse/jetty/client/transport/HttpSender.java

```diff
@@ -201,7 +201,7 @@ protected boolean someToSuccess(HttpExchange exchange)
         };
     }
 
-    private void anyToFailure(Throwable failure)
+    private void anyToFailure(Throwable failure, boolean abort)
```

Comment:
It is a bit strange that this is only called in two places, once with `abort==true` and the other with `abort==false` (which then does it's own abort).

I think it would be better if the method never did the abort and instead returned complete.  The callers could then do their abort.